test/functional/feature_asset_locks.py

```
@@ -758,6 +764,82 @@ def test_v24_fork(self, node_wallet, node, pubkey):
+        # v2 via raw construction
+        lock_tx_v2 = self.create_assetlock(coins.pop(), COIN, pubkey, version=2)
+        self.check_mempool_result(tx=lock_tx_v2,
+            result_expected={'allowed': True, 'fees': {'base': Decimal(str(tiny_amount / COIN))}})
+        self.send_tx(lock_tx_v2)
+
+        # v2 via sendtoaddress (two different platform addresses)
+        platform_addr_from_key = encode_platform_p2pkh('tdash', hash160(pubkey))
+        txid1 = node_wallet.sendtoaddress(platform_addr_from_key, 1.0)
+
+        self.log.info("Test subtractfeefromamount for platform sendtoaddress")
+        fee_test_amount = COIN
```

Flake8 (7.3.0)

[error] local variable 'fee_test_amount' is assigned to but never used

(F841)